Ratings_MVP/Ratings_MVP/Services/NetworkService.swift

```diff
@@ -0,0 +1,73 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+import SwiftyJSON
+
+/// Service with intetnet request
+final class NetworkService: NetworkServiceProtocol {
+    // MARK: Constants
+
+    public enum Constants {
+        static let startUrlWithId = "https://api.themoviedb.org/3/movie/"
+        static let endUrlWithId = "?api_key=74448ef651b5d6b0af58f8899305190d&language=en-US"
+        static let popularUrlText =
+            "https://api.themoviedb.org/3/movie/popular?api_key=74448ef651b5d6b0af58f8899305190d&language=en-US&page=1"
+        static let topRatingUrlText =
+            "https://api.themoviedb.org/3/movie/top_rated?api_key=74448ef651b5d6b0af58f8899305190d&language=en-US&page=1"
+        static let upcomingUrlText =
+            "https://api.themoviedb.org/3/movie/upcoming?api_key=74448ef651b5d6b0af58f8899305190d&language=en-US&page=1"
+        static let resultsText = "results"
+        static let baseImageIRLString = "https://image.tmdb.org/t/p/w500"
+    }
+
+    // MARK: Private Properties
+
+    internal let session = URLSession.shared
```

Comment:
```suggestion
    internal let session = URLSession.shared
```
интерна можно и не писать)